sdk/nodejs/api/client.gen.ts

```diff
@@ -217,12 +217,12 @@ export type ID = string
 export type Platform = string
 
 export type ClientContainerOpts = {
-  id?: ContainerID | Container
+  id?: Container
   platform?: Platform
 }
 
 export type ClientDirectoryOpts = {
-  id?: DirectoryID | Directory
+  id?: Directory
```

Comment:
This should be `DirectoryID`.

Reply:
Exact. There is some logic that needs to be duplicated to avoid creating the type altogether.

Reply:
I'm having second thought on it. I think that making this `id` optional is confusing and we should have a dedicated `client.emptyDirectory()` method instead.
Same for optional `id` for `client.socket()`. I don't even know what an empty socket would mean. Same for the `client.container()` which defaults to `scratch`.

Reply:
Done.